ui/src/app/layout/components/header/header.component.ts

```diff
@@ -110,5 +110,7 @@ export class HeaderComponent implements OnInit {
         }
     }
 
-
+    navigateToDashboard() {
+        void this.router.navigate(['/dashboard']);
```

Comment:
Why "void"?

Reply:
With out void there was a warning (Promise returned from navigate is ignored )